lib/popcorn.ex

```diff
@@ -12,12 +12,11 @@ defmodule Popcorn do
   @doc """
   Generates static artifacts to run the project in the browser.
 
-  `out_dir` and `start_module` are mandatory, unless provided
-  via `config.exs`, for example `config :popcorn, out_dir: "static/wasm, start_module: Start"`
+  `out_dir` and `start_module` are mandatory.
 
   Options:
   - `out_dir` - the directory to write artifacts to
-  - `start_module` - a module with `start/0` function, the application's entry point
+  - `start_module` - a module with `start/0` function, used by AtomVM as starting point 
```

Comment:
I'm not sure we should keep this option documented. If we do, let's at least describe what happens by default